R/fallback.R

```diff
@@ -13,34 +13,23 @@
 #' The functions and environment variables on this page control the process.
 #'
 #' @details
-#' Logging and uploading are both opt-in.
-#' By default, for logging, a message is printed to the console
-#' for the first time in a session and then once every 8 hours.
+#' Logging is opt-out but uploading is opt-in.
 #'
-#' The following environment variables control the logging and uploading:
+#' The following functions control the logging and uploading:
 #'
-#' - \code{DUCKPLYR_FALLBACK_COLLECT} controls logging, set it
-#'   to 1 or greater to enable logging.
-#'   If the value is 0, logging is disabled.
-#'   Future versions of duckplyr may start logging additional data
-#'   and thus require a higher value to enable logging.
-#'   Set to 99 to enable logging for all future versions.
-#'   Use [usethis::edit_r_environ()] to edit the environment file.
+#' - To opt the current machine out of logging, run [`fallback_logging_optout()`].
+#'
+#' - To opt in again for logging, run [`fallback_logging_optin()`].
 #'
 #' - \code{DUCKPLYR_FALLBACK_VERBOSE} controls printing, set it
 #'   to \code{TRUE} or \code{FALSE} to enable or disable printing.
 #'   If the value is \code{TRUE}, a message is printed to the console
 #'   for each fallback situation.
 #'   This setting is only relevant if logging is enabled.
 #'
-#' - \code{DUCKPLYR_FALLBACK_AUTOUPLOAD} controls uploading, set it
-#'   to 1 or greater to enable uploading.
-#'   If the value is 0, uploading is disabled.
-#'   Currently, uploading is active if the value is 1 or greater.
-#'   Future versions of duckplyr may start logging additional data
-#'   and thus require a higher value to enable uploading.
-#'   Set to 99 to enable uploading for all future versions.
-#'   Use [usethis::edit_r_environ()] to edit the environment file.
+#' - To opt the current machine in automatic report uploads, run [`fallback_reporting_optin()`].
```

Comment:
It's not the machine, it's the user config:

```suggestion
#' - To opt in to automatic report uploads, run [`fallback_reporting_optin()`].
```